css/styles.css

```
@@ -0,0 +1,86 @@
+/* reset */
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+body {
+  font-family: "Barlow";
+  font-size: 22px;
+  background-color: #f6eee9;
+  font-weight: 500;
+  color: #373737;
+}
+
+h1 {
+  font-size: 48px;
+}
+
+header {
+  background-color: #eee4e0;
+}
+
+img.logo {
+  width: 140px;
+  margin: 12px 22px;
+}
+
+section.introduction {
+  display: flex;
+}
+
+section.introduction div {
+  flex: 1;
+}
+
+div.slogan {
+  padding: 120px 62px 0px 90px;
+}
+
+div.slogan p {
+  margin-top: 18px;
+}
+
+img.illustration {
+  width: 600px;
+}
+
+section.features {
+  padding: 0px 90px;
+}
+
+section.features > div {
```

```suggestion
.features-box {
```

Rola soh usar soh classe nas paradas :)